java/src/org/openqa/selenium/ie/InternetExplorerDriver.java

```diff
@@ -110,15 +111,19 @@ public class InternetExplorerDriver extends RemoteWebDriver {
   public static final String IE_SWITCHES = "ie.browserCommandLineSwitches";
 
   public InternetExplorerDriver() {
-    this(null, null);
+    this(null, null, null);
   }
 
   public InternetExplorerDriver(InternetExplorerOptions options) {
-    this(null, options);
+    this(null, options, null);
   }
 
   public InternetExplorerDriver(InternetExplorerDriverService service) {
-    this(service, null);
+    this(service, null, null);
```

Comment:
Can we please change this to pass the InternetExplorerOptions and the `ClientConfig` object here directly into the 3 args new constructor instead of sending in null values? The same would hold true for the other constructors in this class that are doing the same.